5_databases/week3/queries.sql

```diff
@@ -0,0 +1,105 @@
+-- Active: 1699100471406@@0.0.0.0@3306
+----> Queries
+--> Meal
+SELECT *
+FROM meals
+
+
+INSERT INTO meals (title, description, location, meal_when, max_reservations, price, created_date)
+VALUES ('Shashlyk', 'Pork meat with onions on fire', 'Georgian restaurant', '2023-11-30 18:45:00', '14', '10.00', '2023-11-22')
+
+
+SELECT *
+FROM meals
+WHERE id = 1
+
+UPDATE meals 
+SET title = 'Kebab'
+WHERE id = 6;
+
+
+DELETE 
+FROM meals
+WHERE id = 1;
+
+--> Reservation
+
+SELECT *
+FROM reservations
+
+INSERT INTO reservations (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES (3, 2, '2023-11-28', '555-555-5555', 'Alex', 'alex@example.com');
```

Comment:
it is good that you don't set the id explicitly but instead let the database generate it for you 👍